README.md

```diff
@@ -82,6 +82,35 @@ Prior to starting, make sure you have a Babylon GitHub account and that you have
 - [ios-build-distribution](https://github.com/Babylonpartners/ios-build-distribution)
 - [ios-fastlane-match](https://github.com/Babylonpartners/ios-fastlane-match)
 
+On Babylon's Slack, check the channels iOS Team related channels:
+Public:
+- #ios
+- #ios-build
+- #ios-usa-support
+- #ios-standup
+- #ios-automation
+- #ios-crash-reports
+- #ios-recruiting
+
+Private (ask someone to invite you):
+- ios-underground
+- ios-questions
+- ios-sdk
+- ios-oss
+- ios-recruiting
+
+Others
+- demo_frontend
+- developers
+- general
+- announcements
+
+Apps
+- pull-reminders
+- peakon
+
+And also your squad will have 
```

Comment:
I would re-phrase this: `Make sure you join your Tribe/Squad's Slack channels.`

Reply:
https://github.com/Babylonpartners/ios-playbook/pull/49/commits/2afbc132f6db2fd0b9ef2818b29c8d99c46949c4